touchforms/formplayer/static/formplayer/script/deprecate/entrycontrols_type.js

```diff
@@ -355,9 +355,9 @@ function MultiSelectEntry (args) {
     return function () {
       var cbox = $('#ch-' + i);
       if (cbox.is(':checked')) {
-        cbox.removeAttr('checked');
+        cbox.prop('checked', false);
```

Comment:
Oh, I didn't even notice it was in a deprecate folder. Been doing a lot of grepping.

I'm a little confused about moving things to a deprecated folder. If it turns out that this is still used somewhere, won't moving it break things anyway?